frontend/class-frontend.php

```diff
@@ -1039,6 +1039,15 @@ public function adjacent_rel_links() {
 			return;
 		}
 
+		/**
+		 * Filter: 'wpseo_pre_adjacent_rel_links' - Allows disabling of Yoast adjacent links if this is being handled by other code.
+		 *
+		 * @api bool $links_generated Indicates if other code has handled adjacent links.
+		 */
+		if ( apply_filters( 'wpseo_pre_adjacent_rel_links', false ) ) {
```

Comment:
For better clarity I would also like to ask you to do an explicit return value check like:

```php
if ( true === apply_filters( 'wpseo_disable_adjacent_rel_links', false ) ) {
     return;
}
```
 